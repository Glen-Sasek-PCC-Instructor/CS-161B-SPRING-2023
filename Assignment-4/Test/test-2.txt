9
4
12
3
16
d
21
-1
12
-9
9
16
-1
