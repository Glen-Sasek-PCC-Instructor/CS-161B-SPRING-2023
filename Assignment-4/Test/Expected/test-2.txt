Welcome to my Set of Numbers program!!
Enter numbers for set 1 on separate lines (-1 to end):
>> >> >> >> >> >> Invalid number! Please try again!
>> >> Your set 1 with 6 numbers ordered:
3 4 9 12 16 21
Enter numbers for set 2 on separate lines (-1 to end):
>> >> Invalid number! Please try again!
>> >> >> Your set 2 with 3 numbers ordered:
9 12 16
The new sets are:
Set1: 3 4 21
Set2: 9 12 16
Thank you for checking out my Set of Numbers program!
